src/libexpr/flake/flake.cc

```diff
@@ -140,8 +140,13 @@ static FlakeInput parseFlakeInput(EvalState & state,
                         attrs.emplace(state.symbols[attr.name], (long unsigned int)attr.value->integer);
                         break;
                     default:
-                        throw TypeError("flake input attribute '%s' is %s while a string, Boolean, or integer is expected",
-                            state.symbols[attr.name], showType(*attr.value));
+                        if (state.symbols[attr.name] == "publicKeys") {
```

Comment:
Use the pattern as above to pre-declare the `publicKeys` symbol instead of comparing strings. @roberth this probably makes no significant difference in semantics or anything else right? Here we should just stick with it for consistency.